.github/workflows/enforce-labels.yml

```diff
@@ -7,3 +7,15 @@ on:
 jobs:
   enforce-labels:
     uses: bitwarden/gh-actions/.github/workflows/_enforce-labels.yml@main
+
+  enforce-version-label:
+    if: ${{ !contains(github.event.*.labels.*.name, 'version') }}
+    name: Enforce version label
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check for label
+        run: |
+          echo "PR without the version label cannot be merged."
+          echo "### :x: PR without the version label cannot be merged" >> $GITHUB_STEP_SUMMARY
+          exit 1
```

Comment:
❓  How is this going to affect dependency updates?

Reply:
We need to add `"labels": ["version:<type>"],` to renovate config, and renovate will add the label to every renovate PR.

Reply:
I wonder if that should be `version: patch` for every type of dependency update? What do you think @mimartin12?

Reply:
What happens if one of the dependencies break backwards compatibility?

Reply:
@michalchecinski, at the very least a version should be tied to the dependency. With renovate, it looks like labels can be added, so this just needs to get into the backlog. 
https://docs.renovatebot.com/configuration-options/#labels

Reply:
@mimartin12 That's the feature I referenced above 😃 I've create a task to add the version label to Renovate PRs automatically: https://bitwarden.atlassian.net/browse/BRE-177